examples/json-access/json_access.bal

```diff
@@ -12,38 +12,38 @@ public function main() {
         }
     };
 
-    // Field access is allowed on the `json` typed variable, but the return
+    // Field access is allowed on the `json`-typed variable. However, the return
     // type would be a union of `json` and `error`. An error will be returned
     // either if the value on which the access is done is not a mapping or if
     // it is a mapping, but it does not contain the particular key.
     json|error r1 = j1.fname;
-    // `r1` is of type `json` since `j1` is a mapping and contains the field `fname`.
+    // `r1` is of the type `json` since `j1` is a mapping and contains the field `fname`.
```

Comment:
Doesn't "the" sound odd here?

Reply:
These changes are as suggested by @praneesha in https://github.com/ballerina-platform/ballerina-lang/pull/17102/files#r309103462

Reply:
We need to have "the" before a noun.

Reply:
It just sounds wrong and weird here though. The idea is to say that `r1`'s type is `json`

Reply:
The way it is worded here needs "the".

Reply:
Again, the context matters. In this context, putting "the" here sounds weird and wrong. See https://english.stackexchange.com/questions/57335/a-variable-of-type-int-versus-a-variable-of-an-int-type for example.

Reply:
https://english.stackexchange.com/questions/358223/use-of-the-after-the-prepostion-of

I think we only have to add "the" if there many "json" types and if we have specifically mentioned one before. but we have only one json type. so we don't need "the" IMO

Reply:
It is exactly why we need to have "the" (because we have only one type of JSON).

Reply:
Shall we come to a conclusion on this please? :) I'm OK with either version, but a quick search of something like "of type int" seems to have more results than "of **the** type int".

IMO, the existing form reads OK -  
```
`r1` is of type `json`
```

However, IMO again, if it was phrased as follows, we would need the "the"
```
`r1` is of the `json` type
```

Thoughts please?

Reply:
IMO, we should be going with what's normally used in programming. The first form is what I have seen in computing related literature/articles. Plus reads easy too.

Reply:
We need to have "the" for grammatical correctness.

Reply:
Here "`r1` is of type `json`" looks more natural and correct to me. This is what used in any other technical documentation as well. I think the reason is these are not Standard English, but rather technical terms used commonly. So we shouldn't make it odd by deviating common technical documentations. 

Refer below examples (Search for "of type JSON"). All are official technical documentations.
1. https://dev.mysql.com/doc/refman/5.7/en/json.html
2. https://v1.realworldocaml.org/v1/en/html/handling-json-data.html
3. https://www.postgresql.org/docs/9.4/datatype-json.html
4. https://books.google.lk/books?id=9dh6DAAAQBAJ&pg=PA153&lpg=PA153&dq=%22of+type+json%22&source=bl&ots=6Nyed9xHht&sig=ACfU3U0bpP-JCoPSnSMhgfKChtVNuWAnWg&hl=en&sa=X&ved=2ahUKEwiowPCD1JDkAhXFqI8KHWVjCQEQ6AEwCnoECAkQAQ#v=onepage&q=%22of%20type%20json%22&f=false

Reply:
Thanks all, going ahead without the "the" since that seems to be how it is commonly used.